src/coreclr/jit/gentree.cpp

```diff
@@ -24932,7 +25019,8 @@ GenTree* Compiler::gtNewSimdShuffleNode(
             op2->AsVecCon()->gtSimdVal = vecCns;
 
             // swap the operands to match the encoding requirements
-            retNode =
+            retNode = isAvx10v1 ?
```

Comment:
Looks like further down we're missing handling for `NI_AVX512VBMI_PermuteVar64x8` as `NI_AVX10v1_V512_PermuteVar64x8`

Reply:
I will wait to conclude the discussion https://github.com/dotnet/runtime/pull/101938#discussion_r1607346909

Reply:
resolved with https://github.com/dotnet/runtime/pull/101938#discussion_r1609372267